src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/NetworkAddressChangedTests.cs

```diff
@@ -23,5 +26,56 @@ public void NetworkAddressChanged_JustRemove_Success()
         {
             NetworkChange.NetworkAddressChanged -= _addressHandler;
         }
+
+        [PlatformSpecific(TestPlatforms.Linux)]
+        [Fact]
+        public async void NetworkAddressChanged_AddRemoveMultipleTimes_CheckForLeakingThreads()
```

Comment:
And we should probably put this into outerloop since it has a delay, but that can be done once we figure out the failing `ps`.